templates/pages/assets/operatingsystem.html.twig

```diff
@@ -102,4 +102,10 @@
         __('Serial number'),
         field_options
     ) }}
+
+    {{ fields.htmlField(
+        'install_date',
+        item.fields['install_date'],
```

Comment:
Maybe `|formatted_datetime` filter should be added here, to make the datetime formatted according to user preferences.